baby-gru/src/utils/MoorhenMolecule.ts

```diff
@@ -209,6 +222,42 @@ export class MoorhenMolecule implements moorhen.Molecule {
         return this.drawSymmetry()
     }
 
+    /**
+     * Fetch the matrices representing the biomolecule of the current model from gemmi
+     */
+    fetchBiomoleculeMatrix() : void {
+        if (!this.biomolOn) {
+            this.symmetryMatrices = []
+        } else {
+            this.symmetryMatrices = []
+            const assemblies = this.gemmiStructure.assemblies
+            const n_assembly = assemblies.size()
+            if(n_assembly>0){
+                const assembly = assemblies.get(0)
+                const generators = assembly.generators
```

Comment:
Same as above, `generators.delete()` needs to be called.

Reply:
Done in 98e02208e3be2dfe1892e1d9bf9ac019e067b9d4.